microceph/ceph/osd.go

```diff
@@ -954,27 +954,55 @@ func outDownOSD(osd int64) error {
 	return nil
 }
 
-func safetyCheckStop(osd int64) error {
+func setOsdNooutFlag(set bool) error {
```

Comment:
This might be clearer as two functions:

```
func setOsdNooutFlag() error
func UnsetOsdNooutFlag() error
```

Then we don't need to deal with a boolean argument.

Reply:
That's my initial implementation, but it turns out it's just simple to add a boolean argument to remove the all-most-identical code. Moreover, I think the function name is a bit clear now: "set the osd noout flag" true means set the flag, false means don't set the flag.